lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p2-identity.yaml

```diff
@@ -1294,6 +1339,18 @@ states:
         targetJourney: FAILED
         targetState: FAILED
 
+  MITIGATION_CRI_DWP_KBV:
+    response:
+      type: cri
+      criId: dwpKbv
+    parent: CRI_STATE
+    events:
+      next:
+        targetState: EVALUATE_GPG45_SCORES
```

Comment:
We'll want to handle `access-denied` and `invalid-request` events here too, targeting `MITIGATION_PRE_EXPERIAN_KBV_TRANSITION_PAGE`